roles/rhn-subscription-osd/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Probably need handle subscriptions for osp and osd in single role rhn-subscription:
1: task/main.yml
     - register with RHN
     - call osp.yml for all hosts except ceph_osds
     - call osd.yml for hosts in ceph_osds
2: tasks/osp.yml
     - add pools for OPS (note we need to skip hosts in ceph_osds group)
        Note: we can add re-try (3) with delay of 30 secs between each re-try attempts and get rid of 90 secs pause. This will help with re-runs as well.
     - add OSP repos
3: tasks/osd.yml
     - add pool for OSD (note we need to run this only on hosts in ceph_osds group)
     Note: we can add re-try (3) with delay of 30 secs between each re-try attempts and get rid of 90 secs pause. This will help with re-runs as well.
     - add OSD repos